pkg/app/api/service/pipedservice/service.proto

```diff
@@ -491,8 +491,13 @@ message ReportUnregisteredApplicationConfigurationsResponse {
 }
 
 message CreateDeploymentChainRequest {
+    // Note: The matcher use AND operator to merge all conditions used to determine
+    // which apps should be trigger as node in chain.
     message ApplicationMatcher {
         string name = 1;
+        // The kind is one of: Kubernetes, Terraform, Cloudrun, Lambda, ECS.
```

Comment:
All should be upper case? https://github.com/pipe-cd/pipe/blob/d133bfaf39ed04c5ab16d6bce19df083c031ce11/pkg/model/common.proto#L23-L29

```suggestion
        // The kind is one of: KUBERNETES, TERRAFORM, CLOUDRUN, LAMBDA, ECS.
```

Reply:
Nice catch 👍